src/main/java/core/basesyntax/FigureBase.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+abstract class FigureBase implements FigureArea, FigureDrawable {
+    protected String color;
+
+    public FigureBase(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public abstract double getArea();
+
+    @Override
+    public abstract void getInfo();
```

Comment:
The method name `getInfo` is not quite descriptive of the behavior it should represent. According to the checklist, methods that are conceptually different should not be put into a single interface. If `getInfo` is meant to draw the figure, consider renaming it to `draw` for clarity and to align with the checklist requirements.